app/controllers/identity.js

```diff
@@ -14,16 +14,49 @@ export default class IdentityController extends Controller {
   @tracked isCopyClicked = false;
   @tracked isVerifyClicked = false;
   @tracked profileURL = this.model.profileURL || '';
+  @tracked saveDisabled = true;
+  @tracked generateChainCodeDisabled = this.model.profileURL === undefined;
+  @tracked checkboxDisabled =
+    this.model.profileURL == undefined || this.model.chaincode === undefined;
```

Comment:
`this.model.profileURL == undefined` is already got compared in the above line and result is stored in the `generateChainCodeDisabled` variable, Can we use that variable here, instead of comparing it again here